packages/web-react/src/components/Collapse/demo/CollapseMultipleTriggers.tsx

```diff
@@ -21,7 +21,7 @@ const CollapseMultipleTriggers = () => {
       <Button onClick={toggleHandler} color="secondary">
         Secondary trigger
       </Button>{' '}
-      <ButtonLink onClick={toggleHandler} aria-expanded={isOpen} color="tertiary">
+      <ButtonLink href="#" onClick={toggleHandler} aria-expanded={isOpen} color="tertiary">
```

Comment:
**question (if-minor):** Why are we using ButtonLink instead of the Button? These are not navigable 🤔